### PR TITLE
Implement GetBTC ticker, order book and trades 

### PR DESCRIPTION
**Pull request by Swingcloud**
- What is the purpose of this Pull Request? as title
- What is the related issue for this Pull Request? #138 
- [x] I have added Specs
- [x] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [x] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly

### CODE REVIEW

**Comment by Swingcloud** [comment]
@tmlee after I check all pairs ticker, suddenly found that volume is refer to all BTC trades volume, so I take it out

and the transactions volume I've fixed

File: lib/cryptoexchange/exchanges/getbtc/services/market.rb [hunk 1]
```diff
@@ -28,7 +28,6 @@ def adapt(output, market_pair)
           ticker.last      = NumericHelper.to_d(ticker_data['last_price'])
           ticker.high      = NumericHelper.to_d(ticker_data['max'])
           ticker.low       = NumericHelper.to_d(ticker_data['min'])
-          ticker.volume    = NumericHelper.divide(NumericHelper.to_d(ticker_data['total_btc_traded']), ticker.last)
```
Comment (tmlee) [inline lib/cryptoexchange/exchanges/getbtc/services/market.rb]: @Swingcloud by removing this we wont have any information on volume, is there no way to convert it?

Comment (Swingcloud) [reply lib/cryptoexchange/exchanges/getbtc/services/market.rb]: seems no way, I've check this website, maybe we can do sum up all trades to get the single volume of currency, what do you think?

Comment (tmlee) [reply lib/cryptoexchange/exchanges/getbtc/services/market.rb]: We can use `total_btc_traded`

**Comment by TJ-CoinGecko** [comment]
@tmlee @wongy91  can this proceed? Got another request from GetBTC guys

**Pull request merged**